cmd/mcp-server/eval/judges/remediation_actionable.py

```
@@ -0,0 +1,143 @@
+def _extract_remediation(text):
+    for pattern in (
+        r"(?i)##?\s*remediation[:\s]*(.*?)(?=\n##|\Z)",
+        r"(?i)##?\s*fix[:\s]*(.*?)(?=\n##|\Z)",
+        r"(?i)##?\s*resolution[:\s]*(.*?)(?=\n##|\Z)",
+        r"(?i)##?\s*(?:recommended?\s*)?action[s]?[:\s]*(.*?)(?=\n##|\Z)",
+    ):
+        match = re.search(pattern, text, re.DOTALL)
+        if match and len(match.group(1).strip()) > 10:
+            return match.group(1).strip()
+
+    lines = [l.strip() for l in text.splitlines()
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)